### PR TITLE
Feature subscriptios

### PR DESCRIPTION
**Pull request by RRicardotj**

### CODE REVIEW

File: src/migrations/20200305133048-subscription.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+'use strict'
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('subscription', {
```
Comment (abelosorio) [inline src/migrations/20200305133048-subscription.js]: Let's call this "groupSubscription". The word Subscription makes me think on a GraphQL Subscription (which is something we may use in a future).

File: src/models/subscription.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
+export default (sequelize, DataTypes) => {
+  const Subscription = sequelize.define('subscription', {
+    id: {
+      type: DataTypes.UUID,
+      primaryKey: true,
+      defaultValue: DataTypes.UUIDV4
+    },
+    userId: {
```
Comment (abelosorio) [inline src/models/subscription.js]: No need to define these fields in the model, they are added automatically by Sequelize by the associations you defined.

File: src/schema/resolvers/subscription.js [hunk 3]
```diff
@@ -0,0 +1,10 @@
+import * as subscriptionController from '~/src/controllers/subscription'
+
+const resolvers = {
+  Mutation: {
+    subscribeToGroup: (obj, { input }) => subscriptionController
```
Comment (abelosorio) [inline src/schema/resolvers/subscription.js]: Heads up!

cc'ing @AgustinOsorio to analyze this as a team.

Think about the MVC architecture. In a conventional REST API you'd have:
- The router: The one that knows who is responsible to react to a specific request ("If someone hits THIS url using THIS method (POST, PUT, etc), I'll call to THIS function"). The router *knows nothing* about the controller's signature (what argument it uses or they order).
- The controller: This is responsible to handle a request. It gets, as the only argument, the request information. Frameworks, such as Sequelize, may add extra information for convenience (like the "context" or the response object).
- The model: Defines business logic. Knows NOTHING about the controller.

In brief, lower layers are agnostic from upper layers. They know NOTHING about what happens "above". Ie: the router knows nothing about HTTPS handshake; the controller knows nothing about what route was hit; the model knows nothing about who is making the call to the model.

In your code, your _router_ (the one that takes a request and passes it to its handler) knows too much about the controller: its arguments and they order. You should make the router to know as little as possible about the controller, so you need to simplify this. In this case:

```javascipt
Mutation: {
  subscribeToGroup: subscriptionController.subscribe
}
```

This way, the only thing your router knows is "if someone executes this mutation, I shall call THIS resolver".

I'd highly suggest reading Clean Code (link pinned in the channel #learning-and-sharing of WeDevelop's Slack), and stress on the Single Responsibility Principle.

File: src/migrations/20200305133027-user.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('user', {
+      id: {
+        type: Sequelize.UUID,
+        primaryKey: true,
+        defaultValue: Sequelize.UUIDV4
+      },
+      name: {
```
Comment (AgustinOsorio) [inline src/migrations/20200305133027-user.js]: I think this it should be firstName and LastName.

**Pull request merged**